pkg/ebpftracer/tracer.go

```diff
@@ -315,6 +316,13 @@ func (t *Tracer) ApplyPolicy(policy *Policy) error {
 	for _, eventID := range policy.SystemEvents {
 		t.findAllRequiredEvents(eventID, requiredEventsIDs)
 	}
+	t.log.Debugf("required events: %v", lo.Map(lo.Keys(requiredEventsIDs), func(item events.ID, index int) string {
```

Comment:
Nit: lets wrap this in an `if s.log.IsEnabled(slog.LevelDebug)`. Not the biggest of deals, but still useless work for most installations.